### DISCUSSION TITLE
After rooting the tg799vac vant-r iptv stops working.

### DISCUSSION THREAD

**Discussion by mojja** [hack-technicolor/hack-technicolor#165] [Q&A]
Anyone got a clue why this is? All im doing is following the steps in the guides up to the hardening root access. When i then try to connect the router iptv is not working at all. Im on firmware 17.2.0339.

**Comment by LuKePicci** [comment]
I guess you need the remote management to be kept enabled in order to receive iptv setup.

**Reply by mojja** [reply]
Can it be kept enabled without my router automatically updating or do I need to buy a new router?

**Reply by LuKePicci** [reply]
It could but you might need to make the remote management server think you are on the latest firmware already. It depends on your ISP provisioning scripts. If the script does something like this:
```
.... initial bootstrap provisioning ....

if` firmware is outdated
then 
    push new firmware
    exit
end

.... voip provisioning ....
.... iptv provisioning ....
```
you will need to spoof your current firmware version and avoid provisioning to stop on firmware update. Meanwhile maybe you want to prevent any firmware download request to do nothing and do not mess your rooted setup. This can be done by manually patching the scripts involved in firmware flashing (e.g. /lib/upgrade/platform.sh) in such a way they complete like if new firmware has been successfully installed.

A different approach would be to sniff the provisioning data from your ISP in such a way you can therefore setup it manually, maybe without remote management being kept enabled. I can't do this because I have no active Telia subscriptions but that's certainly doable if you know how to setup stunnel/socat on your vant-r and use them as proxies for cwmpd..
